### DISCUSSION TITLE
Set Termux terminal character width programmatically?

### DISCUSSION THREAD

**Discussion by nathaneltitane** [termux/termux-app#2512] [General]
Can the termux terminal width be set programmatically, via shell script or command?

**Comment by ixenion** [comment]
Also wondering

**Comment by ixenion** [comment]
OK there is key combo: "CTRL ALT +" to increase font and "CTRL ALT -" to decrease.
Every modern android has built in utility called "input" (works only from root for me)
Which can simulate key combo press like so


input keycombination 113 57 157
where 113 CTRL_LEFT, 57 ALT_LEFT, 157 NUMPAD_ADD

(taken from here: https://www.temblast.com/ref/akeyscode.htm , need to translate from hex to dec)

**Comment by ixenion** [comment]
And how to get current width height I think you manage yourself.
